package.json

```diff
@@ -61,6 +61,7 @@
     "versionup:patch": "lerna version patch --conventional-commits --no-git-tag-version",
     "versionup:minor": "lerna version minor --conventional-commits --no-git-tag-version",
     "versionup:major": "lerna version major --conventional-commits --no-git-tag-version",
+    "versionup:prerelease": "lerna version --conventional-commits --no-git-tag-version --conventional-prerelease --force-publish",
```

Comment:
Do we need to add `postversionup:prerelease` too?

Reply:
I'll add the postscript when releasing the Lumos

Reply:
`postversionup`’s purpose is auto-commit
`versionup` will generate changelog.md, but I found that there is no changelog.md in the current directory, if you want to keep it, you need to undo commit to delete changeelog.md after each `versionup`, I think it is better not to auto-commit, so I didn't add it.